### DISCUSSION TITLE
Daily 26/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#297] [Daily]
¿Qué hice ayer?
¿Qué hice o haré hoy?
Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer realicé algunas de las tareas que me asigné. Además continué con la revisión de la documentación y con la corrección del diagrama de clases. También estuve revisando tareas de otros compañeros. Por la tarde participé de la clase de proyecto y por la noche de la clase de Inglés.
Hoy voy a hacer una tarea que me asigné ayer y algunas pocas cosas más, ya que tengo otros temas de los cuales ocuparme. No creo que pueda estar en  la clase de hoy, así que luego veré la grabación.

**Comment by BraianTroncoso** [comment]
Ayer logré realizar muchas tareas. Estuve asignandomé tareas que no se lograron hacer y adelantando lo más que pueda dado el poco tiempo que queda. Es mucho trabajo para los pocos que participamos, espero poder llegar.
Unas de las que logré terminar la tarea de actualizar perfil, me llevó demasiado tiempo pero por fin pude. Y también hoy(recien ahora) la de cargar Imagen, aun no pude hacer que la almacene en el back.
